build-system/server.js

```diff
@@ -486,6 +487,12 @@ app.use('/a4a(|-3p)/', function(req, res) {
     // This is a special case for testing. `localhost` URLs are transformed to
     // `ads.localhost` to ensure that the iframe is fully x-origin.
     adUrl = urlPrefix.replace('localhost', 'ads.localhost') + adUrl;
+    const inaboxParam = 'inabox=1';
```

Comment:
took another look, actually i think we should always use amp4ads-v0.js for `/a4a(|-3p)/`